libpod/oci_conmon_linux.go

```diff
@@ -1704,6 +1704,8 @@ func httpAttachTerminalCopy(container *net.UnixConn, http *bufio.ReadWriter, cid
 	buf := make([]byte, bufferSize)
 	for {
 		numR, err := container.Read(buf)
+		logrus.Debugf("Read fd(%d) %d/%d bytes for container %s", int(buf[0]), numR, len(buf), cid)
```

Comment:
We should drop the debugfs here, I think - this is a pretty hot loop and could be expensive even if it compiles down to a single conditional branch